core/templates/components/ck-editor-helpers/ck-editor-copy-content.service.ts

```diff
@@ -127,7 +127,9 @@ export class CkEditorCopyContentService {
     let elementTagName = (
       containedWidgetTagName || element.tagName.toLowerCase());
     let html = element.outerHTML;
-    html = html.replace(/<!--[^>]*-->/g, '').trim();
+    while (html.includes('<!--')) {
+      html = html.replace(/<!--[^>]*-*-*>*/gm, '');
+    }
```

Comment:
Is there a library we can use to do this or a reference for why this is the correct code? Sanitizing untrusted input is hard to get right and a common security flaw, so I'm wary of rolling this ourselves.

Ditto for other sanitization code below

Reply:
Done